test/integration/docker_test.go

```diff
@@ -158,3 +163,83 @@ func TestForceSystemdEnv(t *testing.T) {
 		validateContainerdSystemd(ctx, t, profile)
 	}
 }
+
+// TestDockerEnvContainerd makes sure that minikube docker-env command is still available even the runtime is containerd
+func TestDockerEnvContainerd(t *testing.T) {
+	if ContainerRuntime() != constants.Containerd {
+		t.Skip("skipping: TestDockerEnvContainerd can only be run with containerd runtime")
+	}
+	profile := UniqueProfileName("dockerenv")
+	//profile := UniqueProfileName("docker-env-containerd")
+	ctx, cancel := context.WithTimeout(context.Background(), Minutes(30))
+	_ = cancel
+	//defer CleanupWithLogs(t, profile, cancel)
```

Comment:
Any reason this is commented out?

Reply:
Restored. My fault, it should not be commented out